ert_gui/main_window.py

```diff
@@ -70,6 +76,18 @@ def __createMenu(self):
         show_about.setMenuRole(QAction.ApplicationSpecificRole)
         show_about.triggered.connect(self.__showAboutMessage)
 
+        help_links = {}
+        with pkg_resources.resource_stream('ert_gui', os.path.join("resources", "gui", "help", "help_links.yml")) as stream:
+            try:
+                help_links = yaml.safe_load(stream)
+            except yaml.YAMLError as exc:
+                print(exc)
```

Comment:
If we bother handling this exception, we should also tell the user what's wrong in understandable terms (something like `the help links configuration files is wrong`.
If instead the help file is just an implementation detail and users are not supposed to mess with it, then maybe we should just let the exception bubble up (and likely crash ert)

Reply:
I like the idea of not failing since the help links are in no way relevant to the functionality of the rest of the program, but i agree the error message should be more descriptive.